mediapipe_api/framework/formats/image_frame.cc

```diff
@@ -21,10 +21,14 @@ MpReturnCode mp_ImageFrame__ui_i_i_ui(mediapipe::ImageFormat::Format format, int
   CATCH_ALL
 }
 
-MpReturnCode mp_ImageFrame__ui_i_i_i_Pui8_PF(mediapipe::ImageFormat::Format format, int width, int height, int width_step, uint8* pixel_data, Deleter* deleter,
+MpReturnCode mp_ImageFrame__ui_i_i_i_Pui8_PF(mediapipe::ImageFormat::Format format, int width, int height, int width_step, uint8* pixel_data,
                                              mediapipe::ImageFrame** image_frame_out) {
```

Comment:
I just noticed that we'll have to change the name of the function too.
In `mp_ImageFrame__ui_i_i_i_Pui8_PF`:
- `ui` means _unsigned int_
- `i` means _int_
- `P` means _pointer_
- `ui8` means `uint8`
- and I assume `F` means _function_.

It represents the types of the arguments.
So the new name should then be:
```suggestion
MpReturnCode mp_ImageFrame__ui_i_i_i_Pui8(mediapipe::ImageFormat::Format format, int width, int height, int width_step, uint8* pixel_data,
                                             mediapipe::ImageFrame** image_frame_out) {
```

Reply:
Have to update MP.NET's side of code if that's the case. How much of an impact of the change would it be?

Reply:
It was already going to impact MP.NET as-is. Though given that it will copy the pixel data internally, it will make things easier. It's just one function that we'll have to change.